src/components/SearchBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import '../styles/search.scss'
+
+export const SearchBar = () => {
+    return (
+        <>
+        <div class="nci-header-nav__secondary">
```

Comment:
Are these classes specific to something that we already have in the CSS? Or did they just come in with something else? If the latter, let's just strip them out and we'll re-style later.